### ISSUE TITLE
Resolve junit_family warning

### ISSUE THREAD

**Issue by clacroix12** [red-hat-storage/ocs-ci#1289]
We should follow the instructions in the warning so this stops showing up and we will be compatible if we move to pytest 6.0.

```
venv/lib/python3.6/site-packages/_pytest/junitxml.py:436
  /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/venv/lib/python3.6/site-packages/_pytest/junitxml.py:436: PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
  Add 'junit_family=legacy' to your pytest.ini file to silence this warning and make your suite compatible.
    _issue_warning_captured(deprecated.JUNIT_XML_DEFAULT_FAMILY, config.hook, 2)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Comment by gklein** [comment]
Based on this [issue](https://github.com/pytest-dev/pytest/issues/6265), setting `unit_family=xunit1` in pytest.ini  should resolve this while keeping us compatible with the legacy xunit format

**Comment by gklein** [comment]
Should be fixed with #1493 

Based on [pytest code ](https://github.com/pytest-dev/pytest/blob/d79179a239ac89b318be47c99ff965376d177257/src/_pytest/junitxml.py#L78) legacy and xunit1 are the same.
